docs/resources/samples/email-chart-image.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Email chart image'
```

Comment:
This title is hard to read because it's three words that could either be nouns or verbs.
```suggestion
title: 'Email a chart image'
```